CapabilityStatement/CapabilityStatement-England.xml

```diff
@@ -0,0 +1,201 @@
+		</resource>
+		<resource>
+			<type value="EpisodeOfCare"/>
+			<profile value="https://fhir.hl7.org.uk/StructureDefinition/UKCore-EpisodeOfCare"/>
+		</resource>
+		<resource>
+			<type value="FamilyMemberHistory"/>
+			<profile value="https://fhir.hl7.org.uk/StructureDefinition/UKCore-FamilyMemberHistory"/>
+		</resource>
+		<resource>
+			<type value="Flag"/>
+			<profile value="https://fhir.hl7.org.uk/StructureDefinition/UKCore-Flag"/>
+		</resource>
+		<resource>
+			<type value="HealthcareService"/>
+			<profile value="https://fhir.hl7.org.uk/StructureDefinition/UKCore-HealthcareService"/>
+		</resource>
+		<resource>
+			<type value="ImagingStudy"/>
+			<profile value="https://fhir.hl7.org.uk/StructureDefinition/UKCore-ImagingStudy"/>
+		</resource>
+		<resource>
+			<type value="Immunization"/>
+			<profile value="https://fhir.hl7.org.uk/StructureDefinition/UKCore-Immunization"/>
+		</resource>
+		<resource>
+			<type value="List"/>
+			<profile value="https://fhir.hl7.org.uk/StructureDefinition/UKCore-List"/>
+		</resource>
+		<resource>
+			<type value="Location"/>
+			<profile value="https://fhir.hl7.org.uk/StructureDefinition/UKCore-Location"/>
+		</resource>
+		<resource>
+			<type value="Medication"/>
+			<profile value="https://fhir.hl7.org.uk/StructureDefinition/UKCore-Medication"/>
+		</resource>
+		<resource>
+			<type value="MedicationAdministration"/>
+			<profile value="https://fhir.hl7.org.uk/StructureDefinition/UKCore-MedicationAdministration"/>
+		</resource>
+		<resource>
+			<type value="MedicationDispense"/>
+			<profile value="https://fhir.hl7.org.uk/StructureDefinition/UKCore-MedicationDispense"/>
+		</resource>
+		<resource>
+			<type value="MedicationRequest"/>
+			<profile value="https://fhir.hl7.org.uk/StructureDefinition/UKCore-MedicationRequest"/>
+		</resource>
+		<resource>
+			<type value="MedicationStatement"/>
+			<profile value="https://fhir.hl7.org.uk/StructureDefinition/UKCore-MedicationStatement"/>
+		</resource>
+		<resource>
+			<type value="MessageHeader"/>
+			<profile value="https://fhir.hl7.org.uk/StructureDefinition/UKCore-MessageHeader"/>
+		</resource>
+		<resource>
+			<type value="Observation"/>
+			<profile value="https://fhir.hl7.org.uk/StructureDefinition/UKCore-Observation"/>
+			<supportedProfile value="https://fhir.hl7.org.uk/StructureDefinition/UKCore-Observation-LabGroup"/>
+			<supportedProfile value="https://fhir.hl7.org.uk/StructureDefinition/UKCore-Observation-Lab"/>
+			<supportedProfile value="https://fhir.hl7.org.uk/StructureDefinition/UKCore-Observation-VitalSigns"/>
+		</resource>
+		<resource>
+			<type value="OperationOutcome"/>
+			<profile value="https://fhir.hl7.org.uk/StructureDefinition/UKCore-OperationOutcome"/>
+		</resource>
+		<resource>
+			<type value="Organization"/>
+			<profile value="https://fhir.hl7.org.uk/StructureDefinition/UKCore-Organization"/>
+		</resource>
+		<resource>
```

Comment:
theres an update on the UKcore one, with OrganizationAffiliation (draft) in it, not essential now, but will be worth adding in a future update